scripts/chat_rendering.py

```diff
@@ -188,14 +189,14 @@ def integrateAtt(rec):
               <source src="{0}" type="{1}">
               <p><a href="{0}"></a> </p>
             </audio>
-            """.format(rec["content-type"])
+            """.format(rec['file-path'], rec["content-type"])
```

Comment:
This doesn't seem right. There are two variables that need to be filled here. This will fail on execution of this line!